### PR TITLE
Bp 11 fix navigation and contact info

### PR DESCRIPTION
**Pull request by jambao24**
5. Create a pull request from that feature branch into master

6. Assign me and Summer to it to review

https://github.com/MinhLu12/bergin-prosthodontics/pull/16

### CODE REVIEW

File: src/scss/shared.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Background colors
+$primary-background-color: #67697C !important;
+$secondary-background-color: #C6878F !important;
+
+// Texts
+$primary-text: white !important;
```
Comment (MinhLu12) [inline src/scss/shared.scss:6]: I think you accidentally took this change when you pulled. The `global.scss` file replaced this, so this `shared.scss` isn't used anymore. Let's delete it.

File: package-lock.json [hunk 2]
```diff
@@ -4,630 +4,8 @@
   "lockfileVersion": 1,
```
Comment (MinhLu12) [inline package-lock.json:4]: Since this is near-modified code, can you add this file to the `.gitignore`?

**Comment by MinhLu12** [comment]
The build is still broken. If you run `npm ci` then you'll see the same issue. 

I haven't looked into it too much, but maybe delete the package-lock.json, re-run npm ci and see if it's autogenerated and works?

**Comment by jambao24** [comment]
> The build is still broken. If you run `npm ci` then you'll see the same issue.
> 
> I haven't looked into it too much, but maybe delete the package-lock.json, re-run npm ci and see if it's autogenerated and works?

Deleted the package-lock.json file, got this error message

james@LAPTOP-RDO80FHV MINGW64 ~/OneDrive/Documents/GitHub/bergin-prosthodontics (BP-11_FixNavigationAndContactInfo)
$ npm ci
npm WARN prepare removing existing node_modules/ before installation
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.

**Comment by MinhLu12** [comment]
My guess is you directly modified the `package-lock.json` file, but we're never suppose to. I think it's a matter of reverting it back to its normal state (what it was before).

**Comment by MinhLu12** [comment]
Okay, I tried this and it worked:
1. Delete the package-lock.json
2. Delete the node_modules
3. Run an `npm install`
4. This should restore the package-lock.json. Run an `npm ci` to check.

As something out of this, can you put up a pull request for the `README` to describe that we should only really use `npm ci` in the build process to retain the dependency graph, and `npm install` to install specific dependencies?

**Pull request merged**